libs/sdk-ui-dashboard/src/plugins/customizer.ts

```diff
@@ -350,6 +350,121 @@ export type FluidLayoutCustomizationFn = (
     customizer: IFluidLayoutCustomizer,
 ) => void;
 
+/**
+ * Set of functions you can use to customize the export layout of the dashboard rendered.
+ *
+ * @alpha
+ */
+export interface IExportLayoutCustomizer<TWidget> {
+    addTransformer(fn: SectionSlidesTransformer<TWidget>): IExportLayoutCustomizer<TWidget>;
+}
+
+/**
+ * @alpha
+ */
+export type ExportLayoutCustomizationFn = <TWidget>(
+    layout: IDashboardLayout,
+    customizer: IExportLayoutCustomizer<TWidget>,
+) => void;
+
+/**
+ * @alpha
+ */
+export type SectionSlidesTransformer<TWidget> = (
+    section: IDashboardLayoutSection<TWidget>,
+    fn: SectionSlidesTransformerFunction<TWidget>,
+) => IDashboardLayoutSection<TWidget>[] | undefined;
+
+/**
+ * @alpha
+ */
+export interface SectionSlidesTransformerFunction<TWidget> {
+    /**
+     * Default transformer for the section. This is used as default transformation method for the section in
+     * case that no plugin override it. Can be used to provide default transformation for the section in custom
+     * plugin.
+     *
+     * @param section - Section to transform
+     */
+    defaultSectionTransformer: (
```

Comment:
Should we repeat Transformer everywere? (having it on containing interface is probably sufficient?)